README.md

```diff
@@ -2,9 +2,22 @@
 
 Sistema de gerenciamento de estoques, lojas e incentivo ao colaborador
 
-### Como rodar?
+### Como rodar o projeto?
 
 1. Abra a pasta `\estoquei-project` no terminal;
 2. digite `npm install` para instalar as dependências;
 3. Rode o servidor com o comando `npx json-server --watch src/assets/data/database.json`;
 4. Rode o front com o comando `npm start`.
```

Comment:
What do you think about deleting these lines? I think that they're already present at section `Como rodar os testes`

```suggestion
```